ark/analysis/spatial_analysis.py

```diff
@@ -67,15 +67,20 @@ def calculate_channel_spatial_enrichment(dist_matrices_dict, marker_thresholds,
 
     # Subsets the expression matrix to only have channel columns
     all_channel_data = all_data.drop(excluded_colnames, axis=1)
+
+    # this will get refactored once the verification refactor PR gets merged in
+    if not np.all(set(marker_thresholds.iloc[:, 0]) == set(all_channel_data.columns)):
```

Comment:
Lets add a brief error check in spatial_analysis_test for this as well

Reply:
Cool, I basically took the last test in for `calculate_channel_spatial_enrichment` and added a little bit on to that.